scripts/publish.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o pipefail
```

Comment:
```suggestion
set -euo pipefail
```

* `-e`: make the script exit if any command here exits with a non-zero exit code
  * _explanation is oversimplified, but close enough_
* `-u`: error out if you try to use an undefined variable